admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts

```diff
@@ -52,6 +52,7 @@ export default class PositionExtension {
   }
 
   /**
+   * @todo: Why this extension requires grid in constructor and as well in extend method? needs clean up.
```

Comment:
I wonder if I should care about this now or try to refacto this in dedicated PR?

Reply:
Deleted the todo. Can be fixed in dedicated PR